.github/CODEOWNERS

```diff
@@ -478,6 +478,7 @@
 /pkg/network/tracer/*usm*.go                    @DataDog/universal-service-monitoring
 /pkg/network/tracer/*_windows*.go               @DataDog/windows-kernel-integrations
 /pkg/network/usm/                       @DataDog/universal-service-monitoring
+/pkg/usm/                               @DataDog/universal-service-monitoring
```

Comment:
we shouldn't have different packages
you should move it to be as a subpackage under pkg/network/usm